### ISSUE TITLE
[QUESTION] Importing a multilingual model from huggingface into COMET?

### ISSUE THREAD

**Issue by lfritzke-mitre** [Unbabel/COMET#73]
I've been doing some basic experimentation with COMET, but I ran into a wall when I tried to use a different XLM-R model, I couldn't quite find any flags to indicate a custom XLM-R model loaded from a local file. 

I know you can train your own metrics with COMET, but could you also use a domain-specific model without compromising the quality of the output?

**Comment by ricardorei** [comment]
Hi @lfritzke-mitre I am not sure I understood it right? do you want to train on top of a custom XLM-R model?

If so you need to change the configs file used to train the model, more specifically you need to change the [`pretrained_model`](https://github.com/Unbabel/COMET/blob/master/configs/models/regression_metric.yaml#L11) and also set the [`encoder_model`](https://github.com/Unbabel/COMET/blob/master/configs/models/regression_metric.yaml#L10) to XLM-RoBERTa.

After training, you can pass it to the `comet-score` CLI or just load it using the `load_from_checkpoint` function.

**Comment by lfritzke-mitre** [comment]
Aha, this mostly answers my question!

To clarify, I want to use the default metrics provided by COMET (WMT20 etc.) in conjunction with a custom pretrained model. This would be a domain-specific XLM-R model, so would the default COMET metrics be appropriate, or would it be best to train a new one?

Just to make sure I have this right, I would keep the YAML file of the metric the same, but change `pretrained_model` to a local path to some XLM-R model? Would it be the folder or a specific file within the model?

Apologies for any confusions.

**Comment by ricardorei** [comment]
Hmmm, I am not sure this will work. I think the best option is to train a new one. 

All COMET models finetune the underlying XLM-R model. This means that the top layers (like the Feed Forward Estimator) rely on the representations coming from the underlying XLM-R which is also updated during training. If you change the encoder for something else, with different representations, this will affect the quality predictions in an unexpected way.

If you believe that your domain-specific model provides more quality representations than the original XLM-R I would recommend you to train a new COMET model possibly by keeping your XLM-R encoder frozen during training.

I hope this was helpful

**Comment by lfritzke-mitre** [comment]
This answers my question perfectly, thanks!
